app/services/ElasticsearchPlugin.scala

```diff
@@ -837,73 +822,86 @@ class ElasticsearchPlugin(application: Application) extends Plugin {
     builder
   }
 
-  /**Convert search string into an Elasticsearch-ready JSON query object**/
+  /** Convert search string into an Elasticsearch-ready JSON query object **/
   def prepareElasticJsonQuery(query: String, permitted: List[UUID], user: Option[User]): XContentBuilder = {
-    /** OPERATORS
-      *  ==  equals (exact match)
-      *  !=  not equals (partial matches OK)
-      *  <   less than
-      *  >   greater than
-      **/
 
     // Use regex to split string into a list, preserving quoted phrases as single value
     val matches = ListBuffer[String]()
-    val m = Pattern.compile("([^\"]\\S*|\".+?\")\\s*").matcher(query.replace(":", " "))
+    val m = Pattern.compile("([^\"]\\S*|\".+?\")\\s*").matcher(query)
```

Comment:
I noticed that white space between key, operator and value is not allowed. Could we add that so that the user can have extra white space such as:
"syn:loc_selector"  ==   "5310:123-456-7890"